src/scp/LocalNode.cpp

```diff
@@ -381,8 +380,8 @@ LocalNode::findClosestVBlocking(SCPQuorumSet const& qset,
 Json::Value
 LocalNode::toJson(SCPQuorumSet const& qSet, bool fullKeys) const
 {
-    return toJson(qSet, [&](NodeID const& k)
-                  { return mDriver.toStrKey(k, fullKeys); });
+    return toJson(
+        qSet, [&](NodeID const& k) { return mDriver.toStrKey(k, fullKeys); });
```

Comment:
weird that it decided to change format here?

Reply:
I don't really know what is going on with `clang-format`. Every version does things differently 🤷 
I *guess* the logic here is, put all arguments on the same line if possible, but if not, the body of a lambda is a nice cutting point. Since the change reduced the length of the line, it probably went back to putting all args on the same line.